src/Core/src/Hosting/MauiAppBuilder.cs

```diff
@@ -104,6 +109,12 @@ ILoggingBuilder InitializeLogging()
 			}
 		}
 
+		IDictionary<object, object> IHostApplicationBuilder.Properties => throw new NotImplementedException();
```

Comment:
Not sure what this `Properties` represents, but it may just be a temporary dictionary. So, this may be simple.

> Gets a central location for sharing state between components during the host building process.

Reply:
Yes, this is just a simple in-memory dictionary for consumers to store state on the builder.

Here's a usage example:

https://github.com/dotnet/aspire/blob/16953c4e71f6af49a6af671899a4eef754510580/src/Components/Common/HealthChecksExtensions.cs#L25-L30